### ISSUE TITLE
OrientedPathFollow should be renamed PathFollowOriented

### ISSUE THREAD

**Issue by aaronfranke** [godotengine/godot#23070]
**Godot version:** 3.1 Alpha and master since May 25

**Issue description:** See https://github.com/godotengine/godot/pull/18752

The new OrientedPathFollow should be renamed PathFollowOriented, so that all of the Path nodes start with the word Path. This node isn't present in 3.0 so renaming doesn't break compatibility.

However, it might even be better for PathFollow to have an "Oriented" boolean or enum instead.

Suggested Milestone: 3.1. A decision should be made on this issue before 3.1 Beta.

**Comment by danilo2205** [comment]
1 - Yes, I think it's a good idea to rename it.

2 - The oriented boolean idea is good as well, altho it would add runtime checks (i.e. every update the bool property would be checked to correctly manage its behaviour.).

3 - I tried to use inheritance at first but later I figured I wouldn't be using "rotation_mode" at all, so it was kind of pointless to inherit it.

**Comment by reduz** [comment]
Indeed, just make it as a boolean in PathFollow, I dont even understand why two different nodes were created for this in the first place
